distributor-api-v1/operations.md

```diff
@@ -166,6 +170,10 @@ Preferred initial call used to obtain all static data about distributor configur
 | `Theme` | string | optional | TBC |
 | `VoucherCode` | string | optional | TBC |
 
+#### Requiredfield
```

Comment:
@dobras This is inconsistent with the name of the type in the table, should be `RequiredField`.